examples/src/main/java/tachyon/examples/BasicOperations.java

```diff
@@ -120,14 +121,14 @@ private boolean readFile(TachyonFileSystem tachyonFileSystem, long fileId)
   }
 
   public static void main(String[] args) throws IllegalArgumentException {
-    if (args.length != 3) {
+    if (args.length != 4) {
       System.out.println("java -cp " + Constants.TACHYON_JAR
           + " tachyon.examples.BasicOperations <TachyonMasterAddress> <FilePath> "
```

Comment:
BasicOperations.class.getName() + " <master address> <file path> <tachyon storage type for writes (STORE | NO_STORE)> <under storage type for writes (SYNC_PERSIST | NO_PERSIST)>"